scripts/pysamstats

```diff
@@ -205,8 +218,10 @@ Version: {version} (pysam {pysamversion})
                 window_size=options.window_size,
                 window_offset=options.window_offset,
                 min_mapq=options.min_mapq,
-                no_dup=options.no_dup
+                no_dup=options.no_dup,
+                stepper=options.stepper
```

Comment:
Not 100% sure here. I think stepper should be always passed to kwargs, but would be worth checking.

Reply:
It may be OK, but I think strictly the stepper should only apply when doing pileup-based stats. So maybe stepper should get moved below under the check for ``if not options.type.endswith('_binned'):``.

Reply:
Yep, CI is failing with "TypeError: stat_binned() got an unexpected keyword argument 'stepper'", so I guess this needs to get moved.